youtube_dl/downloader/external.py

```diff
@@ -181,7 +181,8 @@ def _call_downloader(self, tmpfilename, info_dict):
 
         args = [ffpp.executable, '-y']
 
-        args += self._configuration_args()
+        if self.exe == self.get_basename():
```

Comment:
> This looks like a hack for me.
it's clearly an inconsistency.

it's simplest way to fix this problem, the names can be changed to make them more clear, for example:
`get_basename` -> `exe_name`
`exe` -> `selected_external_downloader`
and the code can be moved into `_configuration_args` to make it general.

for a better way to pass `external-downloader-args` in the config, i will ask @jaimeMF if it possible to it to #8901.